draft-ietf-quic-http.md

```diff
@@ -799,25 +799,34 @@ indicated request.  This trades off network usage against a potential latency
 gain.  HTTP/3 server push is similar to what is described in HTTP/2 {{?HTTP2}},
 but uses different mechanisms.
 
-Each server push is identified by a unique Push ID. This Push ID is used in one
-or more PUSH_PROMISE frames (see {{frame-push-promise}}) that carry the request
-fields, then included with the push stream which ultimately fulfills those
-promises. When the same Push ID is promised on multiple request streams, the
-decompressed request field sections MUST contain the same fields in the
-same order, and both the name and the value in each field MUST be exact
-matches.
-
-Server push is only enabled on a connection when a client sends a MAX_PUSH_ID
-frame; see {{frame-max-push-id}}. A server cannot use server push until it
-receives a MAX_PUSH_ID frame. A client sends additional MAX_PUSH_ID frames to
-control the number of pushes that a server can promise. A server SHOULD use Push
-IDs sequentially, starting at 0. A client MUST treat receipt of a push stream
-with a Push ID that is greater than the maximum Push ID as a connection error of
-type H3_ID_ERROR.
+Each server push is identified by a unique Push ID, which is used to refer to
```

Comment:
It might be beneficial to state that it is the server that assigns Push IDs to pushes (even though it is not only that server that can use Push IDs to refer to pushes).